src/main/resources/static/test-scheme.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE Article (
+  id INTEGER NOT NULL IDENTITY,
+  title VARCHAR(45) NOT NULL,
+  label VARCHAR(45) NOT NULL,
```

Comment:
it is not the best idea to use 45 size for varchar. On the background MySQL or other data providers will use base size like: 8, 16, 32, 64, 128, 256, etc. So for the future pls use them